draft-looker-jwm.xml

```diff
@@ -534,18 +537,20 @@
 
       <t>Support for digitally signed JWMs using JWS is REQUIRED. Of the signature and 
       MAC algorithms specified in JSON Web Algorithms <xref target="RFC7518"/>, only 
-      ECDSA using the P-256 curve and SHA-256 hash algorithm ("ES256") MUST be implemented 
+      ECDSA using the Ed25519 curve and SHA-512 hash algorithm MUST be implemented 
```

Comment:
These were correct the way you had them.

Reply:
I realised there is a typo with ECDSA because the edward curves use EdDSA, but Im actually suggesting a change so that we are using the edwards based curves for digital signatures by default and X25519 for key agreement, @troyronda @llorllale do you guys have opinions on this?